tutorials/irpa-cai-integration/irpa-cai-integration.md

```diff
@@ -0,0 +1,381 @@
+
+    !![overview](step1-0.png)
+
+3.  From the pop up, check the **Name** and **Version Number** and click **Generate Package**.
+
+    !![generate-package](step1-1.png)
+
+4.  Navigate to **Packages** tab in Cloud Factory to view the generated package. Search for the package **Orders Management**.
+
+5.  Click the drop-down. From the line-item generated, click **...** and select **Release** to release the package.
+
+    !![release package](step1-2.png)
+
+6.  In the **Release Package** pop-up, click **Release**. The package will be in **Released** status.
+
+    !![release package confirmation](step1-3.png)
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Create the Trigger and API Key)]
+This step involves creating a trigger of type API and an API Key in your environment. A trigger of type API opens a dedicated endpoint that allows an external application to start the execution of an automation or scenario in a specified deployed package via an HTTP POST call. As such, an API trigger is always attached to one automation or scenario in one deployed package. A deployed package is bound to an environment, which means that using an API trigger will guarantee job execution on one of the agents in the environment of the deployed package.
+
+1.  Click **...** against the released version and select **Create Trigger**.
+
+    !![Create trigger](step2-1.png)
+
+2.  From the Create Trigger pop-up, select an environment and click **Next**.
+
+    !![create trigger step1](step2-2.png)
+
+3.  In **Step 2 Deploy**, package will be deployed automatically and in **Step 3 Set Environment Variables**, the variables will be set, if any.
+    In **Step 4 Select a trigger type**, select the radio button **API** and click **Next**.
+
+    !![Trigger Type API](step2-3.png)
+
+4.  In the **Create API trigger** pop-up, provide the following information:
+
+    |  Field          | Value
+    |  :------------- | :-------------
+    |  Name           | `API_CAI_Call`
+    |  Description    | An API triggered from CAI chatbot
+    |  Execute          | Collect Order Details
+    |  Priority   | Medium
+    |  Job Expiration | 60 Minutes
+
+    !![create API trigger](step2-4.png)
+
+5.  The API trigger information will be generated. Click **Copy** and paste it in notepad. Click **Close**.
+    The newly generated trigger is visible in your Environment.
+
+    !![API URL generated](step2-5.png)
+
+6.  Select **API Keys** in your Environment and click **Add API Key**. Input the **Name** as `myAPIKey` and click **Next**.
+
+    !![Create API Key](step2-6.png)
+
+7.  Activate `environment_read`, `trigger_read` and `trigger_execute` and click **Next**.
+
+    !![Runtime options](step2-7.png)
+
+8.  Review your API Key and click **Add**.
+
+9.  Click **Copy** and make sure to note down the key since you will not be able to view it again.
+
+    !![API key](step2-8.png)
+
+The API trigger and API Key are created successfully.
+
+[VALIDATE_7]
```

Comment:
Why do you start with VALIDATE_7? The first validation in a tutorial is always VALIDATE_1